templates/register.html

```diff
@@ -0,0 +1,17 @@
+{% extends '_layout.html' %}
+
+{% block body %}
+<h2 class="mb-5">Login to Your account</h2>

```

Comment:
same thing as the other heading

Reply:
is it just headings or do we want all text fields (like First Name, Email) to be lowercase as well?

Reply:
idk i meant to talk abt that last week but we'll talk abt it during next sprint or retro